src/scorer.py

```diff
@@ -6,6 +6,7 @@
 from collections import Counter
 
 from .objdump import ArchSettings, Line, objdump, get_arch
+from colorama import Fore
```

Comment:
I don't think we should add a dependency on colorama just for this. Do like main.py and hard-code color codes

Reply:
makes sense